components/post_emoji/index.jsx

```diff
@@ -16,13 +16,16 @@ function mapStateToProps(state, ownProps) {
     const emoji = emojiMap.get(ownProps.name);
 
     let imageUrl = '';
-    let displayTextOnly = false;
+    let displayTextOnly = true;
```

Comment:
Should this really default to true? I'm pretty sure the desired behaviour is to show a blank placeholder before the emoji is loaded (or not loaded) and not the text. This also makes the `else if` below useless as all it would be doing is setting `displayTextOnly` to true when it already was true

Reply:
Thanks, let me back up again on this and revisit the changes I made.